packages/inngest/src/components/connect/index.ts

```diff
@@ -141,6 +141,12 @@ class WebSocketWorkerConnection implements WorkerConnection {
     if (!Array.isArray(options.handleShutdownSignals)) {
       options.handleShutdownSignals = DEFAULT_SHUTDOWN_SIGNALS;
     }
+
+    const env = allProcessEnv();
+    options.signingKey = options.signingKey ?? env[envKeys.InngestSigningKey];
+    options.signingKeyFallback =
+      options.signingKeyFallback ?? env[envKeys.InngestSigningKeyFallback];
```

Comment:
Nit: This should probably be `||` instead of `??` so that we look at the environment if someone passes an empty key like `signingKey: ""`.
```suggestion
    options.signingKey = options.signingKey || env[envKeys.InngestSigningKey];
    options.signingKeyFallback =
      options.signingKeyFallback || env[envKeys.InngestSigningKeyFallback];
```

Reply:
Good point!